app-header/app-header.html

```diff
@@ -256,6 +262,16 @@
           type: Boolean,
           reflectToAttribute: true,
           value: false
+        },
+
+        /**
+         * Snaps the header to the top of the scrolling region when the header is revealed.
+         * To use this property, the header should have the `reveals` property as well.
+         */
+        snaps: {
+          type: Boolean,
+          reflectToAttribute: true,
```

Comment:
Does this need `reflectToAttribute`? I don't see any style rules for `[snaps]`...